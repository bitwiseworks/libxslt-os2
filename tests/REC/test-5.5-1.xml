<?xml version="1.0" encoding="UTF-8"?>
<winelist>
  <wine grape="Chardonnay">
    <winery>Benziger</winery>
    <product>Carneros</product>
  </wine>
</winelist>
