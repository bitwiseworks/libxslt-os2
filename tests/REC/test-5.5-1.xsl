<?xml version="1.0" encoding="UTF-8"?>
<!-- transformation of the calloutlist element -->
<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">

  <xsl:template match="text()">
  </xsl:template>

  <xsl:template match="@*|node()">
    <xsl:copy>
      <xsl:apply-templates/>
    </xsl:copy>
  </xsl:template>

</xsl:stylesheet>
