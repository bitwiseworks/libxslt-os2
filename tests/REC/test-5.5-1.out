<?xml version="1.0"?>
<winelist>
  <wine>
    <winery>Benziger</winery>
    <product>Carneros</product>
  </wine>
</winelist>
